### PR TITLE
[WIP] Refactor python's helper module

### PR DESCRIPTION
**Pull request by dmitry-shibanov**
- Merged Archive-ToolZip and ZipTool
- renamed functions
- remove New-CompleteFile

### CODE REVIEW

File: helpers/nix-helpers.psm1 [hunk 1]
```diff
@@ -41,8 +39,16 @@ Function Download-Source {
     } catch {
         "$_"
         break
-    }
-    
+    }    
+}
+
+Function UnPack-TarArchive {
```
Comment (maxim-lobanov) [inline helpers/nix-helpers.psm1]: nitpick: Unpack

File: helpers/win-helpers.psm1 [hunk 2]
```diff
@@ -28,31 +28,21 @@ function Install-App {
 function Zip-Tool {
```
Comment (maxim-lobanov) [inline helpers/win-helpers.psm1]: `Archive-SevenZip`

Comment (dmitry-shibanov) [reply helpers/win-helpers.psm1]: Done.

File: helpers/nix-helpers.psm1 (hunk 1 above)
Comment (dmitry-shibanov) [reply helpers/nix-helpers.psm1]: Done.

**Pull request merged**